injector/network_disruption.go

```diff
@@ -309,12 +309,15 @@ func (i *networkDisruptionInjector) Clean() error {
 //   - a first prio qdisc will be created and attached to root
 //     it'll be used to apply the first filter, filtering on packet IP destination, source/destination ports and protocol
 //   - a second prio qdisc will be created and attached to the first one
-//     it'll be used to apply the second eBPF filter, filtering on method and path
+//     it'll be used to apply the second eBPF filter, filtering on method
 //   - a third prio qdisc will be created and attached to the second one
+//     it'll be used to apply the second eBPF filter, filtering on path
+//   - a fourth prio qdisc will be created and attached to the second one
```

Comment:
Isn't it attached to the third one, in the diagram?

Reply:
🙈 Indeed...